style.css

```diff
@@ -109,3 +129,199 @@ p {
 .logo2 {
   padding: 20px 20px 20px 0;
 }
+
+.header2 {
```

Comment:
- [ ] As required, I kindly suggest that you ensure the main container of your works section has a display of grid.